ddl/ddl_api.go

```diff
@@ -1030,9 +1030,15 @@ func (d *ddl) CreateView(ctx sessionctx.Context, s *ast.CreateViewStmt) (err err
 	if !ok {
 		return infoschema.ErrDatabaseNotExists.GenWithStackByArgs(ident.Schema)
 	}
-	if is.TableExists(ident.Schema, ident.Name) {
+	oldView, err := is.TableByName(ident.Schema, ident.Name)
```

Comment:
Should we check whether `oldView.IsView` is true here?

Reply:
+1.

Reply:
Done